jfrog-applications/jfrog-cli/cli-for-jfrog-artifactory.md

```diff
@@ -2094,6 +2094,18 @@ jf dotnet restore --build-name=my-build-name --build-number=1
 
 In addition, record the build-info as part of build **my-build-name/1**. The build-info can later be published to Artifactory using the [build-publish](https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli/cli-for-jfrog-artifactory#publishing-build-info) command.
 
+##### Example 4
+
+To incorporate both the artifact and its dependency within a module in a build, utilize the --module flag with the following commands:
```

Comment:
To incorporate both the artifact and its dependency within a module in a build, utilize the --module flag with the following commands:
-->
To incorporate both the build artifacts and the dependencies within the same module in the published build-info, utilize the **--module** option with the following commands: